common/i18n/resources/en.js

```diff
@@ -256,6 +256,7 @@ module.exports = {
   },
 
   formErrors: {
+    atLeastOneInvalidValue: 'At least one invalid value',
```

Comment:
Can the message be something like : one or more values are invalid. What do you think?